xcpmd/src/acpi-events.c

```diff
@@ -40,14 +40,6 @@ static struct event acpi_event;
 
 static struct ev_wrapper ** acpi_event_table;
 
-void adjust_brightness(int increase, int force) {
-    (void) increase;
-    if (force || (pm_quirks & PM_QUIRK_SW_ASSIST_BCL) || (pm_quirks & PM_QUIRK_HP_HOTKEY_INPUT)) {
-        xcpmd_log(LOG_WARNING, "Brightness adjustment not implemented yet.");
```

Comment:
... and it's removed here anyway.

Reply:
The DBus call for vglass is added back in `handle_bcl_event`.